pilot/pkg/networking/core/cluster_waypoint.go

```diff
@@ -57,9 +58,32 @@ func buildInternalUpstreamCluster(name string, internalListener string) *cluster
 	}
 }
 
+func buildEncapInternalUpstreamCluster(name string, internalListener string) *cluster.Cluster {
```

Comment:
This seems identical to buildInternalUpstreamCluster. is it?

Reply:
Yeah this is from a former iteration where I had a custom filter. I'll clean this up once I get this all working locally